templates/common/.devcontainer/devcontainer.json/csharp/devcontainer.json

```diff
@@ -19,6 +19,9 @@
         "ghcr.io/devcontainers/features/node:1": {
             "version": "16",
             "nodeGypDependencies": false
+        },
+        "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1":{
+            "version":"latest"
         }
```

Comment:
We recently merged merged #1210. This means even with bicep installed, `azd` will still acquire it's own copy of `bicep`. Thus, we can simply revert this change.